validate_request.py

```diff
@@ -357,7 +357,7 @@ def validate_latlon_in_bboxes(lat, lon, bboxes, coverages=[]):
     for bbox in bboxes:
         valid_lat = bbox[1] <= lat <= bbox[3]
         valid_lon = bbox[0] <= lon <= bbox[2]
-        if valid_lat or valid_lon:
+        if valid_lat and valid_lon:
```

Comment:
Btw, this is a bug I found/fixed while working on this PR. I'm pretty sure I am responsible for this bug and failed to fix it the same time these other two lines were fixed:

https://github.com/ua-snap/data-api/blob/5b4d056dc20eeb5e5c536510340f2f77c4acb75f/validate_request.py#L99

https://github.com/ua-snap/data-api/blob/5b4d056dc20eeb5e5c536510340f2f77c4acb75f/validate_request.py#L130

Obviously, both the provided latitude and longitude need to be valid to be inside the coverage's BBOX! 🤦 This fix was necessary to get the API to show me the coverage extent error page using the `invalid_latlon_outside_coverage.html` template when an invalid lat/lon is provided.

Reply:
Well, I looked at the git blame for that and it was on a PR that I reviewed! So, right there with you 😀 
Good catch here.